src/test/groovy/org/prebid/server/functional/tests/FilterMultiFormatSpec.groovy

```diff
@@ -0,0 +1,382 @@
+package org.prebid.server.functional.tests
+
+import org.prebid.server.functional.model.bidder.Generic
+import org.prebid.server.functional.model.config.AccountAuctionConfig
+import org.prebid.server.functional.model.config.AccountConfig
+import org.prebid.server.functional.model.config.PreferredMediaType
+import org.prebid.server.functional.model.db.Account
+import org.prebid.server.functional.model.request.auction.Audio
+import org.prebid.server.functional.model.request.auction.Banner
+import org.prebid.server.functional.model.request.auction.BidRequest
+import org.prebid.server.functional.model.request.auction.Bidder
+import org.prebid.server.functional.model.request.auction.Native
+
+import static org.prebid.server.functional.model.response.auction.ErrorType.GENERIC
+import static org.prebid.server.functional.model.response.auction.MediaType.AUDIO
+import static org.prebid.server.functional.model.response.auction.MediaType.BANNER
+
+class FilterMultiFormatSpec extends BaseSpec {
+
+    def "PBS should response with all requested media type when default adapters multi format true in config and at account level specified preferred media type"() {
```

Comment:
"PBS should response with all requested media type when default adapters multi format true in config and at account level specified preferred media type" -> "PBS should respond with all requested media types when default adapters multi format is true in config and preferred media type specified at account level"

There is a very similar grammar errors in all test names. Please proofread and correct them.